### PR TITLE
Improve documentation about Nomad ACL minimum rights

### PR DESCRIPTION
**Pull request by Thadir**
### What does this PR do?

Add missing information in documetation about what richt traefic needs in a nomad ACL setup

### Motivation

Not having this makes the documetation very incompleet and the fact its documeted in a issue is ver very dumb

### More

- [X] Added/updated documentation

### CODE REVIEW

File: docs/content/providers/nomad.md [hunk 1]
```diff
@@ -163,6 +163,7 @@ providers:
 _Optional, Default=""_
 
 Token is used to provide a per-request ACL token, if Nomad ACLs are enabled.
+The ACL rights that you should apply to this token is 'read-job' as described here in the [nomad documentation about ACL](https://developer.hashicorp.com/nomad/tutorials/access-control/access-control-policies)
```
Comment (jspdown) [inline docs/content/providers/nomad.md:166]:
```suggestion
The appropriate ACL privilege for this token is 'read-job', as outlined in the [Nomad documentation on ACL](https://developer.hashicorp.com/nomad/tutorials/access-control/access-control-policies).
```

**Review by jspdown — CHANGES_REQUESTED** [review]
Thanks for opening this PR. Just one minor suggestion:

**Comment by Thadir** [comment]
> Thanks for opening this PR. Just one minor suggestion:

Processed it its not my main lanaguage so any sugetsion is welcome.

**Review by kevinpollet — APPROVED** [review]
Thanks 👍

**Review by rtribotte — APPROVED** [review]
Thanks 👍

**Review by jspdown — APPROVED** [review]

**Pull request merged**